backend/start_server.py

```
@@ -2,39 +2,44 @@
         requests.get("http://localhost:11434/api/tags", timeout=5)
         print("✅ Ollama is running!")
         return True
-    except request.exceptions.ConnectionError:
+    except request.exceptions.RequestException:
         print("❌ Ollama is not running or is running in another port")
         print("Trying to start Ollama...")
 
-        # Start Ollama in the background
-        subprocess.Popen(["ollama", "serve"], sdout=subprocess.DEVNULL, stderr=subprocess.DEVLNULL)
+        subprocess.Popen(
+            ["ollama", "serve"],
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)